course_info/static/course_info/js/controllers/EnrollmentsController.js

```diff
@@ -0,0 +1,117 @@
+(function() {
+    var app = angular.module('CourseInfo');
+    app.controller('EnrollmentsController', EnrollmentsController);
+
+    function EnrollmentsController($scope, $routeParams, courseInstances, $compile, djangoUrl) {
+        $scope.course_instance_id = $routeParams.course_instance_id;
+        var ci = courseInstances.instances[$scope.course_instance_id];
+        if (angular.isDefined(ci)) {
```

Comment:
Nope.  What I'm doing here is handling the case where I've reloaded the page while on the people view...in that case, `ci` will be undefined (the `courseInstances.instances` object will be completely empty), and I need to pull it in via ajax.